core/src/main/java/org/infinispan/persistence/file/SingleFileStore.java

```diff
@@ -141,13 +140,13 @@ public void init(InitializationContext ctx) {
    public void start() {
       try {
          file = getStoreFile(ctx.getGlobalConfiguration(), configuration.location(), ctx.getCache().getName());
-         if (!file.exists()) {
+         if (!SecurityActions.fileExists(file)) {
             File dir = file.getParentFile();
             if (!dir.mkdirs() && !dir.exists()) {
```

Comment:
Both File.mkdirs() and File.exists() can throw SecurityException.

Reply:
Updated to use a security action for mkdirs too @pferraro, but I still haven't added a test :)